packages/snowpack/src/commands/install.ts

```diff
@@ -43,19 +44,15 @@ import {
   findMatchingAliasEntry,
 } from '../util.js';
 
+const logger = createLogger({name: 'snowpack'});
```

Comment:
Passing a `name` in is how logs are prefixed with `[snowpack]`. This can be changed for plugins (e.g. `[@snowpack/plugin-babel] …`).